platform/android/export/export.cpp

```diff
@@ -2656,6 +2655,83 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	Error get_cl(const Ref<EditorExportPreset> &p_preset, const String &p_path, int p_flags, Vector<uint8_t> *clf) {
+
+		bool use_32_fb = p_preset->get("graphics/32_bits_framebuffer");
+		bool immersive = p_preset->get("screen/immersive_mode");
+		bool debug_opengl = p_preset->get("screen/opengl_debug");
+		String cmdline = p_preset->get("command_line/extra_args");
+		int version_code = p_preset->get("version/code");
+		String version_name = p_preset->get("version/name");
+		String package_name = p_preset->get("package/unique_name");
+
+		Vector<String> cl = cmdline.strip_edges().split(" ");
+		for (int i = 0; i < cl.size(); i++) {
+			if (cl[i].strip_edges().length() == 0) {
+				cl.remove(i);
+				i--;
+			}
+		}
+
+		gen_export_flags(cl, p_flags);
+
+		bool apk_expansion = p_preset->get("apk_expansion/enable");
+		String apk_expansion_pkey = p_preset->get("apk_expansion/public_key");
+		if (apk_expansion) {
+
```

Comment:
Taking a peek around https://github.com/godotengine/godot/blob/master/platform/android/export/export.cpp I don't see cases where there is an extra newline at the start of blocks and functions. Please remove these extra newlines (including code you already submitted).